lib/Doctrine/DBAL/Migrations/Tools/Console/Command/GenerateCommand.php

```diff
@@ -127,4 +136,30 @@ protected function generateMigration(Configuration $configuration, InputInterfac
 
         return $path;
     }
+
+    protected function manageCustomTemplate(Configuration $configuration, InputInterface $input, OutputInterface $output)
+    {
+        if ($tpl = $input->getOption('template')) {
```

Comment:
Using the null coalesce operator is only supported by php 7. Avoid using it for php 5.6 should be a good thing.